/*
 * ConfigInc.h
 *
 *  Created on: 2016-8-22
 *      Author: Ralf
 */

#ifndef CONFIGINC_H_
#define CONFIGINC_H_

#include "Demo.pb.h"
#include "ConfigBuilding.pb.h"
#include "CommonGiftConfig.pb.h"
#include "UserConfig.pb.h"
#include "ConfigItem.pb.h"
#include "ConfigProductLine.pb.h"
#include "ConfigOrder.pb.h"
#include "ConfigShop.pb.h"
#include "ConfigAd.pb.h"
#include "ConfigTask.pb.h"
#include "ConfigVIP.pb.h"
#include "ConfigShipping.pb.h"
#include "ConfigAlliance.pb.h"
#include "ConfigNPCSeller.pb.h"
#include "ConfigReward.pb.h"
#include "ConfigFriend.pb.h"
#include "LevelupUnlock.pb.h"
#include "ProtoNPCUser.pb.h"
#include "ConfigAssistor.pb.h"

#include "ConfigMapExpand.pb.h"
#include "ConfigAllianceRace.pb.h"
#include "ConfigNPCCustomer.pb.h"
#include "ConfigRandomBox.pb.h"
#include "ConfigMailDog.pb.h"
#include "ConfigRotaryTable.pb.h"
#include "ConfigFriendlyTree.pb.h"
#include "ConfigActivityTencent.pb.h"
#include "ConfigAccessAd.pb.h"
#include "ConfigTheme.pb.h"

#include "ConfigActivity.pb.h"
#include "ConfigKeeper.pb.h"
#include "ConfigCard.pb.h"
#include "ConfigLanguage.pb.h"
#include "ConfigFriendWorker.pb.h"
#include "ConfigPet.pb.h"
#include "CdKey.pb.h"
#endif /* CONFIGINC_H_ */
